### DISCUSSION TITLE
Error while evaluating property 'compilerOptions.jvmTarget' of task ':capacitor-community-bluetooth-le:compileDebugKotlin

### DISCUSSION THREAD

**Discussion by minorityuk** [capacitor-community/bluetooth-le#568] [General]
Hi

I have just added the plugin to my ionic 7 app and when I try to run the command ionic capacitor run android -l --external -debug  I am getting the following error reported below,  Strangely if I go into Android studio and run debug from there the app does appear on my physical device so I am not sure why Capacitor command is having an issue.  I am running latest version of Android Studio but just can't seem to understand the issue and I have not touched my gardle.build file.

[capacitor]         * What went wrong:
[capacitor]         Execution failed for task ':capacitor-community-bluetooth-le:compileDebugKotlin'.
[capacitor]
[capacitor]         > Error while evaluating property 'compilerOptions.jvmTarget' of task ':capacitor-community-bluetooth-le:compileDebugKotlin'.
[capacitor]         > Failed to calculate the value of property 'jvmTarget'.
[capacitor]         > Unknown Kotlin JVM target: 20

**Comment by lajith111** [comment]
I am not sure..but seems i had get same errror.. i think u need to remove  targetapi in manfest file

In android folder..
